resources/fetchersManager/factory/factory.go

```diff
@@ -76,15 +76,26 @@ func getAwsConfig(
 		return awssdk.Config{}, nil, nil
 	}
 
-	awsConfig, err := awsCfgProvider.InitializeAWSConfig(ctx, cfg.CloudConfig.AwsCred)
-	if err != nil {
-		return awssdk.Config{}, nil, fmt.Errorf("failed to initialize AWS credentials: %w", err)
+	var awsConfig awssdk.Config
+	var err error
+	if cfg.Benchmark == config.CIS_AWS {
+		// initialize AWS config without setting the region to be the ec2 metadata region
```

Comment:
Perhaps expand the comment to explain why we want to do that, it's not really clear to me from this snippet

Reply:
I'll elaborate, thx.